Board Meeting Minutes/CCC Board Minutes 2020-03-12.md

```diff
@@ -0,0 +1,159 @@
+**Governing Board Conference Call -- 7:00am PST**
+
+**12 March 2020**
+
+1.  **Call to Order / Roll Call**
+
+    1.  **Premier Members of the CCC**
+
+        1.  Philippe Robin (Arm)\*
+
+        2.  Sergio Leunissen (Oracle)\*
+
+        3.  Mike Bursell (Red Hat)\*
+
+        4.  Dave Thaler (Microsoft)\* (TAC Chair)
+
+        5.  Ron Perez (Intel)\*
+
+        6.  Michael Cheng (Facebook)\*
+
+        7.  Gilad Golan (Google)\*
+
+        8.  Peixin Hou (Huawei)\*
+
+    2.  **Guests / Observers**
+
+        1.  Richard Searle (Fortanix, General Member Rep)\*
+
+        2.  Seth Knox (Outreach Chair)
+
+        3.  Simon Johnson (Intel)
+
+        4.  Stephano Cetola (Linux Foundation, Acting Secretary)
+
+    3.  **Not in attendance**
+
+        1.  Xiaoning Li (Alibaba)\* 
+
+        **\*voting member**
+
+2.  **Agenda Items & Motions**
+
+    1.  The board approves the minutes of the February 27, 2020 meeting
+        with no abstentions or objections.
+
+    2.  The board approves the Outreach expense for Linux Foundation
+        Creative Design Support Services at a 'PLUS' level for an annual
+        cost of \$15,000. No abstentions or objections. It should be
+        noted that the budget subcommittee chair and budget committee
+        member brought this to the board with the Outreach chair
+        present.
+
+3.  **Action Item Review**
+
+    1.  \[Stephen\] User subcommittee is under discussion. \[in
+        progress\]
+
+    2.  \[Dave\] is working with Morgan on the TAC (JPMC). \[done\]
+
+    3.  \[Stephen\] Bringing clarity to the different stages of new
+        project approval (TAC vs GB with regards to questions regarding
+        trademark search. \[in progress\]
+
+    4.  \[Stephen/Stephano\] Start the process of the F2F in Austin.
+        \[done\]
+
+4.  **Code of conduct**
+
+    1.  Stephano will create a distribution list which will be closed to
+        voting board members (<conduct@confidentialcomputing.io>) for
+        now, and will likely be changed in the future.
+
+    2.  The code of conduct will be updated via pull requests on GitHub.
+        TAC will head up those changes.
+
+    3.  The TAC is advised to recommend projects use the Contributor
+        Covenant 2.0, and if they want to use another code of conduct,
+        they should provide that document and highlight the differences.
+        This will be proposed to the board next meeting to vote on.
+
+    4.  As projects receive code of conduct complaints, they should
+        notify the reflector so that the CCC can stay informed of these
+        issues. This will also be brought up in the next governing board
+        meeting for a vote.
+
+    5.  The suggestion was made that we start all our meetings with a
+        Code of Conduct disclaimer. Stephen will research that and get
```

Comment:
Stephen is not listed in the roll call of these minutes, was he present?

Reply:
No, he was not. I've called out Philippe as the acting chair for that meeting, good point.